tests/Transports.AspNetCore.Tests/WebSockets/ReusableMemoryReaderStreamTests.cs

```diff
@@ -10,6 +10,8 @@ public ReusableMemoryReaderStreamTests()
         _stream = new ReusableMemoryReaderStream(_buffer);
     }
 
+    public void Dispose() => _stream.Dispose();
```

Comment:
This is correct … but does it matter? The GC will catch it. These are just tests. We are not creating 1000 of these instances.

Reply:
Whether this change has a value here/now or not, it is not important. The code should be correct, regardless of whether it is from test or not. The analyzers helped to find the lost call and it's good.

Reply:
No, I’m sorry but this is ridiculous and unnecessary. These are tests. You write them and move on. 

Did you even look at the code here? There is no implementation of the dispose method. This method does not do ANYTHING. Even if disposal mattered.

Reply:
The point is not what method does now or not, but to write the code so as to minimize possible problems, no matter how they arise.

> ridiculous and unnecessary

In the same way, we can answer the use of analyzers in the main code for some cases, but this is not a reason not to use them. I do not want to keep in my head (and I can’t) that something is not necessary in this place, but it in other does, and selectively apply the rules depending on what I now consider important and what is not.

This change does not harm the code, but brings its quality a bit closer to the main code.